client/common/cmdline.c

```diff
@@ -1349,6 +1366,12 @@ int freerdp_client_settings_command_line_status_print_ex(rdpSettings* settings,
 			settings->ListMonitors = TRUE;
 		}
 
+		arg = CommandLineFindArgumentA(largs, "smartcard-list");
+		if (arg->Flags & COMMAND_LINE_VALUE_PRESENT)
+		{
+			settings->ListSmartcards = TRUE;
```

Comment:
Done like with `ListMonitors` but not really sure it is a good thing to add such flags in `rdpSettings`. Can´t we just use a return code instead?

Reply:
Yeah I was not comfortable with that either, because I would have liked the listSmartcard to take in account the passed parameters (like the CSP or the reader), but I've not find a way to do that easily.